src/server/items/controllers.js

```diff
@@ -1,27 +1,64 @@
 import db from '../database';
 import {APIError} from '../ErrorHandler';
-import {KeyMaker, DEFAULT_PRECISON} from './models';
+import {KeyMaker, KeyUtils, ALIVE, DEFAULT_PRECISON} from './models';
 
 export default {
   getAll: (req, res, cb) => {
```

Comment:
Any recommendations? haha
I thought basically this function is used for querying all items,
and if you want to add some conditions you can add querystring for that. :)